sysutils/scala_select/Portfile

```diff
@@ -1,25 +1,26 @@
-PortSystem 1.0
-PortGroup select 1.0
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup select    1.0
 		
-name            scala_select
-version         0.1
-revision        2
-categories      sysutils
-platforms       darwin
-license         BSD
-maintainers     nomaintainer
-supported_archs noarch
-description     common files for selecting default scala compiler
-long_description \
-    scala_select installs files that allow 'port select' to switch the \
-    default scala compiler. It creates unsuffixed symlinks in the \
-    MacPorts prefix that point to the selected version.
+name                scala_select
+version             0.1
+revision            3
+categories          sysutils
+platforms           darwin
+license             BSD
+maintainers         nomaintainer
+supported_archs     noarch
+description         common files for selecting default scala compiler
+long_description    scala_select installs files that allow 'port select' to switch the \
+                    default scala compiler. It creates unsuffixed symlinks in the \
+                    MacPorts prefix that point to the selected version.
 
-homepage        https://www.macports.org/
+homepage            https://www.macports.org/
 distfiles
 
-use_configure   no
-build           {}
+use_configure       no
+build               {}
```

Comment:
You can simplify this port, via use of pg `stub`:
* Include that first, followed by pg `select`.
* Change phase `destroot` to `post-destroot`.
* Eliminate `platforms`, `license`, `maintainers`, and `homepage`.
* Eliminate `use_configure`, as well as phase `build`.

Reply:
Also be sure to run `port lint --nitpick` on all of your ports, to catch issues like the following:

```
port lint scala_select:
Warning: Line 5 contains tab but modeline says tabs should be expanded
```

Reply:
@mascguy thanks!